tests/MSBuildDeviceIntegration/Tests/DebuggingTest.cs

```diff
@@ -302,13 +302,24 @@ public override void OnCreate ()
 				/* embedAssemblies */    false,
 				/* fastDevType */        "Assemblies:Dexes",
 			},
+			new object[] {
+				/* useSharedRuntime */   true,
+				/* embedAssemblies */    false,
+				/* fastDevType */        "Assemblies:Dexes",
+			},
+			new object[] {
+				/* useSharedRuntime */   true,
+				/* embedAssemblies */    false,
+				/* fastDevType */        "Assemblies:Dexes",
```

Comment:
The `Assemblies;Dexes` mode will not change the apk when an AndroidResource is altered. So I'm not sure this is a good test. We should just use the `Assemblies` mode. That way any resource change will end up in the apk.

Reply:
Updated